aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -96,55 +96,94 @@ Consider using the [Azure SignalR Service](xref:signalr/scale#azure-signalr-serv
 
-* `Program.cs`:
+To provision the Azure SignalR Service for an app in Visual Studio:
 
-  ```csharp
-  builder.Services.AddSignalR().AddAzureSignalR(options =>
-  {
-      options.ServerStickyMode = 
-          Microsoft.Azure.SignalR.ServerStickyMode.Required;
-  });
-  ```
+1. Create an Azure Apps publish profile in Visual Studio for the app.
+1. Add the **Azure SignalR Service** dependency to the profile. If the Azure subscription doesn't have a pre-existing Azure SignalR Service instance to assign to the app, select **Create a new Azure SignalR Service instance** to provision a new service instance.
+1. Publish the app to Azure.
 
-* Configuration (use **one** of the following approaches):
+Provisioning the Azure SignalR Service in Visual Studio automatically adds the SignalR connection string to the app service's configuration.
 
-  * In `appsettings.json`:
+## Azure App Service without the Azure SignalR Service
 
-    ```json
-    "Azure:SignalR:ServerStickyMode": "Required"
-    ```
+:::moniker range=">= aspnetcore-8.0"
 
-  * The app service's **Configuration** > **Application settings** in the Azure portal (**Name**: `Azure__SignalR__ServerStickyMode`, **Value**: `Required`). This approach is adopted for the app automatically if you [provision the Azure SignalR Service](#provision-the-azure-signalr-service).
+Hosting a Blazor Web App that uses interactive server-side rendering on Azure App Service requires configuration for Application Request Routing (ARR) affinity and WebSockets. The App Service should also be appropriately globally distributed to reduce UI latency. Using the Azure SignalR Service when hosting on Azure App Service isn't required.
 
-> [!NOTE]
-> The following error is thrown by an app that hasn't enabled sticky sessions for Azure SignalR Service:
->
-> > blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.
+:::moniker-end
 
-### Provision the Azure SignalR Service
+:::moniker range="< aspnetcore-8.0"
 
-To provision the Azure SignalR Service for an app in Visual Studio:
+Hosting a Blazor Server app on Azure App Service requires configuration for Application Request Routing (ARR) affinity and WebSockets. The App Service should also be appropriately globally distributed to reduce UI latency. Using the Azure SignalR Service when hosting on Azure App Service isn't required.
 
-1. Create an Azure Apps publish profile in Visual Studio for the app.
-1. Add the **Azure SignalR Service** dependency to the profile. If the Azure subscription doesn't have a pre-existing Azure SignalR Service instance to assign to the app, select **Create a new Azure SignalR Service instance** to provision a new service instance.
-1. Publish the app to Azure.
+:::moniker-end
+
+1. Configure the app for session affinity by setting the Azure SignalR `ServerStickyMode` option to `Required`. Typically, an app creates the configuration using **one** of the following approaches:
```

Comment:
This bit describes config steps for using the Azure SignalR Service, but using the Azure SignalR Service isn't required or even recommended when hosted on Azure App Service anymore. You do need to configure session affinity when deploying to Azure App Service, which is done by enabling the "ARR affinity" setting: https://learn.microsoft.com/azure/app-service/configure-common#configure-general-settings